_sass/_main.scss

```diff
@@ -13,6 +13,10 @@
   @include padded-container();
 }
 
+.centered {
```

Comment:
If we're going to use this as an `extend`, we may want to move it into its own partial in the `extends` folder so that it can be loaded earlier. This way, it is available to all partials instead of just `_main.scss`.